test/index.html

```diff
@@ -23,8 +23,7 @@
 				'../helpers/test/dom.html',
 				'../helpers/test/focus.html',
 				'../helpers/test/contrast.html',
-				'../mixins/test/arrow-keys-mixin.html',
-				'../templates/primary-secondary/test/primary-secondary.html'
```

Comment:
it'll automatically pick up the test file you've added?

Reply:
Yeah Karma is configured to pull in all `*.test.js` files.